File: BackEnd/backend/src/main/java/com/fitapp/backend/routinecomplete/routine/aplication/dto/response/RoutineStatisticsResponse.java
package com.fitapp.backend.routinecomplete.routine.aplication.dto.response;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class RoutineStatisticsResponse {
    @JsonProperty("totalRoutines")
    private long totalRoutines;
    
    @JsonProperty("activeRoutines")
    private long activeRoutines;
    
    @JsonProperty("inactiveRoutines")
    private long inactiveRoutines;
}

File: FrontEnd/app/src/main/java/com/fitapp/appfit/feature/routine/model/rutine/response/RoutineStatisticsResponse.kt
package com.fitapp.appfit.feature.routine.model.rutine.response

import com.google.gson.annotations.SerializedName

data class RoutineStatisticsResponse(
    @SerializedName("totalRoutines") val totalRoutines: Long,
    @SerializedName("activeRoutines") val activeRoutines: Long,
    @SerializedName("inactiveRoutines") val inactiveRoutines: Long
)